lib/ansible_tower_client/inventory_source.rb

```diff
@@ -0,0 +1,21 @@
+module AnsibleTowerClient
+  class InventorySource < BaseModel
+    def self.endpoint
+      "inventory_sources".freeze
+    end
+
+    def can_update?
+      response = api.get("#{url}update").body
```

Comment:
Is `url` guaranteed to end with `/`?  Would `File.join(url, "update")` be safer?